fiftyone/core/sample.py

```diff
@@ -57,13 +57,15 @@ def __getitem__(self, field_name):
         if self.media_type == fomm.VIDEO and fofu.is_frame_number(field_name):
             return self.frames[field_name]
 
+        self._check_reserved_keywords([field_name])
```

Comment:
I don't think we need validation on `__getitem__`. If the data is there, go ahead and let the user access it.